tools/boards.txt.py

```diff
@@ -325,6 +325,23 @@
                   'Product page: https://www.adafruit.com/product/2821'
                   ],
     }),
+    ( 'cw01', {
+        'name': 'XinaBox CW01',
+        'opts': {
+            '.build.board': 'ESP8266_GENERIC',
+            '.build.variant': 'xinabox',
+            },
+        'macro': [
+            'resetmethod_nodemcu',
+            'flashmode_qio',
+            'flashfreq_40',
+            '4M',
+            ],
+        'desc': [ 'The XinaBox CW01 is an Arduino-compatible Wi-Fi development board powered by Ai-Thinker\'s ESP-12F, clocked at 80 MHz at 3.3V logic. It also has an onboard RGB LED.',
```

Comment:
160MHz is always a possible option for any board.
Maybe you would also add a gpio define in the pins/variant file about your RGB led for your users.

Reply:
i will add that info as well

Reply:
any ideas as to why the package build keeps failing ?